src/server.js

```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+import { isEmpty, merge } from 'lodash';
```

Comment:
Elsewhere, functions are imported from lodash using the `lodash/functionName` package:

https://github.com/stellar/js-stellar-sdk/blob/a95dbe61329000b0eee2d342f98b5fb9476f22bb/src/config.js#L1

We should probably do the same here.